dotCMS/src/integration-test/java/com/dotcms/graphql/DotGraphQLHttpServletTest.java

```diff
@@ -0,0 +1,78 @@
+package com.dotcms.graphql;
+
+import static org.junit.Assert.*;
+
+import com.dotcms.mock.request.MockHttpRequest;
+import com.dotcms.mock.response.MockHeaderResponse;
+import com.dotcms.mock.response.MockHttpResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.util.IntegrationTestInitService;
+
+public class DotGraphQLHttpServletTest {
+
+    
+    @BeforeClass
+    public static void prepare() throws Exception{
+
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void testing_cors_headers() {
+        
+        HashMap<String,String> headers = new DotGraphQLHttpServlet().corsHeaders.apply();
+        
+        assertEquals(headers.get("access-control-allow-origin"), "*");
+        assertEquals(headers.get("access-control-allow-credentials"), "true");
+        assertEquals(headers.get("access-control-allow-headers"), "*");
+        assertEquals(headers.get("access-control-allow-methods"), "GET,PUT,POST,DELETE,HEAD,OPTIONS,PATCH");
+        
+        // this property is specifically overriden for graphql
+        assertEquals(headers.get("access-control-expose-headers"), "Content-Type,Cache-Control");
+    }
+
+    @Test
+    public void testing_GETRequestToGraphQLServer_returnResponseWithExpectedHeaders()
+            throws ServletException, IOException {
+
+        MockHttpRequest request = new MockHttpRequest("localhost", "/");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5609581)